File: apps/wow-minmax-ui/src/app/modules/character/model/CharacterSelectionOptions.ts
import { EnemyType } from "../../shared/model/character/EnemyType";
import { LevelDifference } from "../../shared/model/character/LevelDifference";
import { Phase } from "../../shared/model/character/Phase";

export interface CharacterSelectionOptions {
	phases: Phase[];
	enemyTypes: EnemyType[];
	enemyLevelDiffs: LevelDifference[];
	lastModifiedCharacterId: string;
}

File: services/wow-minmax/src/main/java/wow/minmax/model/CharacterSelectionOptions.java
package wow.minmax.model;

import wow.commons.model.character.CreatureType;
import wow.commons.model.pve.Phase;

import java.util.List;

/**
 * User: POlszewski
 * Date: 2025-09-03
 */
public record CharacterSelectionOptions(
		List<Phase> phases,
		List<CreatureType> enemyTypes,
		List<Integer> enemyLevelDiffs
) {
}
